lib/rules/no-test-module-for.js

```diff
@@ -0,0 +1,81 @@
+'use strict';
+
+const types = require('../utils/types');
+const emberUtils = require('../utils/ember');
+const utils = require('../utils/utils');
+
+const ERROR_MESSAGE = 'moduleFor* apis are are not allowed. Use `module` instead of `moduleFor`';
+
+//------------------------------------------------------------------------------
+// Rule Definition
+//------------------------------------------------------------------------------
+
+module.exports = {
+  meta: {
+    docs: {
+      description: 'Disallow use of moduleFor, moduleForComponent, etc',
```

Comment:
Remember to run `yarn update` after changing this description to ensure the README matches.